staker/rollup_watcher.go

```diff
@@ -71,15 +74,21 @@ func (r *RollupWatcher) getCallOpts(ctx context.Context) *bind.CallOpts {
 
 func (r *RollupWatcher) getNodeCreationBlock(ctx context.Context, nodeNum uint64) (*big.Int, error) {
 	callOpts := r.getCallOpts(ctx)
-	createdAtBlock, err := r.GetNodeCreationBlockForLogLookup(callOpts, nodeNum)
-	if err != nil {
+	if !r.unSupportedL3Method.Load() {
+		createdAtBlock, err := r.GetNodeCreationBlockForLogLookup(callOpts, nodeNum)
+		if err == nil {
+			return createdAtBlock, nil
+		}
 		log.Trace("failed to call getNodeCreationBlockForLogLookup, falling back on node CreatedAtBlock field", "err", err)
-		node, err := r.GetNode(callOpts, nodeNum)
-		if err != nil {
-			return nil, err
+		if strings.Contains(err.Error(), "execution reverted") {
```

Comment:
I don't think we actually want to fall back to GetNode here in case of an unknown kind of error, because on an L3 the CreatedAtBlock will be the wrong value (it'll be the L1 block number instead of the parent chain block number)

Reply:
cool